### DISCUSSION TITLE
Include organization ID in getOrganization API call

### DISCUSSION THREAD

**Discussion by gregory-claeyssens** [forem/forem#17565] [Feature Requests]
I've noticed the organization ID isn't included in the [getOrganization](https://developers.forem.com/api/#operation/getOrganization) API call.
While it is expected on the [createArticle](https://developers.forem.com/api/#operation/createArticle) API call.
You can't seem to use the organization username here. The only way to find your organization ID seems to be by going to your organization's analytics dashboard, here you'd find the ID inside the URL.

Am I missing something obvious here?

![Screenshot 2022-05-04 105431](https://user-images.githubusercontent.com/5057847/166650757-3ba4cc5a-7ed9-4e97-98a2-af8646bc8a04.jpg)

**Comment by jeremyf** [comment]
I believe this is now addressed with forem/forem#17600

**Comment by Dante8426** [comment]
How you find the organization name. As we are facing issue with finding the organization name for [huntsville al therapist](https://www.proformancetherapyandwellness.com/hsv-physical-therapy-prime--tune-ups.html) account. But are not finding it anywhere. Could you share the complete guidance along with screenshot so that we could find it and use it for the our working activities.
